app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from typing import List
+
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: List, cafe: Cafe) -> str:
+    """Check if list of friends can go to cafe."""
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
Returning immediately upon catching a `VaccineError` means that the function will stop checking the rest of the friends. If the intention is to check all friends and then decide, consider accumulating errors and deciding after the loop.